monkey/agent_plugins/util.sh

```diff
@@ -33,15 +34,23 @@ lower() {
 get_plugin_filename() {
     _plugin_path=${1:-"."}
 
-    # if "manifest.yaml" doesn't exist, assume the file is called "manifest.yml"
-    # the script will fail if that doesn't exist either
-    if [ ! -f "${_plugin_path}/$MANIFEST_FILENAME" ]; then
-        MANIFEST_FILENAME=manifest.yml
-    fi
+    MANIFEST_FILENAME=$(get_plugin_manifest_filename)
```

Comment:
This is confusing. MANIFEST_FILENAME is already set, and then this resets it. But `get_plugin_manifest_filename` relies on the value already set in MANIFEST_FILENAME. It's a bit spaghetti-ish.